processing/deps/discharge/cross_sectional_discharge.py

```diff
@@ -231,12 +232,18 @@ def left_to_right_discharge_ogr(
         "gauge_line_id": ogr.OFTInteger,
         "q_net_sum": ogr.OFTReal,
     }
+
+    # gridadmin.gpkg is in the same directory as the gridadmin.h5 file
+    gridadmin = gr.grid_file
```

Comment:
Please make "Gridadmin geopackage" a separate argument to the processing algorithm instead of assuming that it is in this directory

Reply:
Added input field for gridadmin.gpkg file in a new commit